.circleci/config.templ.yml

```diff
@@ -141,12 +141,22 @@ commands:
             - start_docker_services:
                 env: SNAPSHOT_CI=1
                 services: testagent << parameters.docker_services >>
+            - run: ulimit -c unlimited
             - run:
                 environment:
                   DD_TRACE_AGENT_URL: << parameters.trace_agent_url >>
                   RIOT_RUN_RECOMPILE_REQS: "<< pipeline.parameters.riot_run_latest >>"
                 command: |
                   ./scripts/run-test-suite '<<parameters.pattern>>' <<pipeline.parameters.coverage>> 1
+            - run:
+                command: |
+                  mkdir -p /tmp/core_dumps
+                  cp core.* /tmp/core_dumps
```

Comment:
In general, is it possible for any of the test processes to modify the cwd so as to move the corefiles elsewhere?  If not, please ignore.  If so, then we probably have limited ability to modify the corepattern etc, but maybe we could do a broader search with `find`

Reply:
Ah this is a good point. I suppose it would be possible for this to happen. We could experiment with trying to set an absolute path just to be sure